src/main/java/me/hydos/blaze4d/api/shader/MinecraftUbo.java

```diff
@@ -91,18 +85,14 @@ public int getSize() {
     public void update(int currentImg, @NotNull Swapchain swapChain) {
         if (uboFrames.isEmpty()) return;
 
-        if (dirty) {
```

Comment:
why remove the dirty check? atleast change the code that calls this function so we don't do any unnecessary ones. ubo updates are one of the largest sources of slowness in the project.

Reply:
I added that in for an idea i had. it wasnt already there